src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/Client/WattTimeClientTests.cs

```diff
@@ -106,7 +106,7 @@ public async Task GetDataAsync_DeserializesExpectedResponse()
 
         var data = await client.GetDataAsync("balauth", new DateTimeOffset(2022, 4, 22, 0, 0, 0, TimeSpan.Zero), new DateTimeOffset(2022, 4, 22, 0, 0, 0, TimeSpan.Zero));
 
-        Assert.IsTrue(data.Count() > 0);
```

Comment:
Any reason why these changes with this PR? Looks like they are not related with the PR goal.

Reply:
Oh these were unintentional, I ran a formatting script that cleaned this up unintentionally. I'll revert